src/pages/Settings.tsx

```diff
@@ -17,22 +19,27 @@ const layout = {
 
 const tailLayout = {
   wrapperCol: {
-    offset: 2,
+    offset: 4,
   },
 };
 
 const Settings: React.FC = () => {
   const [form] = useForm();
 
   useEffect(() => {
+    let adminAPIConfig = getAdminAPIConfig();
```

Comment:
please use ES6 destructing here https://es6.ruanyifeng.com/#docs/destructuring

```js
const {adminAPISchema, adminAPIHost, ....} = getAdminAPIConfig()
```

Reply:
maybe use `schema, host, path, key` instead of `adminAPIxxx`?

Reply:
Although we should use the ES6 standard, this may cause the host or path to conflict with other content in the future.

Reply:
`schema, host, path, key` will be changed to